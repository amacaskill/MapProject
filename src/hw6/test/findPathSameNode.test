# tests output if both nodes entered are the same node 

CreateGraph noEdges
AddNode noEdges char1

FindPath noEdges char1 char1

CreateGraph g
AddNode g char1
AddNode g char2
AddEdge g char1 char2 label

FindPath g char1 char1

FindPath g char2 char2