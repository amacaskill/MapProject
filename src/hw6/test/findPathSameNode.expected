# tests output if both nodes entered are the same node 

created graph noEdges
added node char1 to noEdges

path from char1 to char1:

created graph g
added node char1 to g 
added node char2 to g
added edge label from char1 to char2 in g

path from char1 to char1:

path from char2 to char2: